### ISSUE TITLE
Trigger resend of welcome email via API

### ISSUE THREAD

**Issue by MorrisJobke** [nextcloud/server#3367]
* provide an API endpoint in the OCS API to resend the welcome email for a given user

Best would maybe to include this into the user provisioning API: https://docs.nextcloud.com/server/12/admin_manual/configuration_user/user_provisioning_api.html

Something like 
```
PUT http://admin:secret@example.com/ocs/v1.php/cloud/users/Frank/resendWelcomeMessage
```

@rullzer @nickvergessen @blizzz  Opinions on this?

**Comment by nickvergessen** [comment]
We have a welcome email? Never saw something like that?

**Comment by MorrisJobke** [comment]
> We have a welcome email? Never saw something like that?

It's the password reset email. You can enable this in the user management settings (lower left).

**Comment by MorrisJobke** [comment]
But maybe we just make this a separate email template to enhance the wording.

**Comment by blizzz** [comment]
Totally, a password reset as welcome email sounds rather confusing, may even look like phishing, if it is unexpected.

**Comment by MorrisJobke** [comment]
> Totally, a password reset as welcome email sounds rather confusing, may even look like phishing, if it is unexpected.

Makes sense to have a dedicated welcome email with proper wording then.

**Comment by MorrisJobke** [comment]
Welcome email was brought via #2287

**Comment by MorrisJobke** [comment]
Fix for this is in #3607
